examples/matmul.mojo

```diff
@@ -278,19 +358,28 @@ def main():
     constrained[N % tile_n == 0, "N must be a multiple of tile_n"]()
     constrained[K % tile_k == 0, "K must be a multiple of tile_k"]()
 
+    print("Problem Size (M N K):", M, N, K)
+
     test_all()
     print("CPU Results\n")
     var python_gflops = run_matmul_python()
-    run_matmul_numpy()
+    var numpy_gflops = run_matmul_numpy()
 
     # Don't run all these benchmarks in CI, too resource intensive
     if not getenv("CI"):
-        bench[matmul_naive, "Naive:"](python_gflops)
-        bench[matmul_vectorized, "Vectorized: "](python_gflops)
-        bench[matmul_parallelized, "Parallelized:"](python_gflops)
-        bench[matmul_tiled, "Tiled:"](python_gflops)
-        bench[matmul_unrolled[0], "Unrolled:"](python_gflops)
-        bench[matmul_unrolled[1], "Physical Cores:"](python_gflops)
-        bench[matmul_unrolled[2], "Logical Cores:"](python_gflops)
-    # CHECK: Performance Cores
-    bench[matmul_unrolled[3], "Performance Cores:"](python_gflops)
+        bench[matmul_naive, "Naive:"](python_gflops, numpy_gflops)
+        bench[matmul_vectorized, "Vectorized:"](python_gflops, numpy_gflops)
+        bench[matmul_parallelized, "Parallelized:"](python_gflops, numpy_gflops)
+        bench[matmul_tiled, "Tiled:"](python_gflops, numpy_gflops)
+        bench[matmul_unrolled[0], "Unrolled:"](python_gflops, numpy_gflops)
+        bench[matmul_unrolled[1], "Unrolled - Physical Cores:"](
```

Comment:
Can the formatting of these please be changed back to original to keep the result in line e.g. `Physical Cores:`